ci_framework/roles/ci_network/tasks/apply.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: "Apply network {{ network.key }}"
+  become: true
+  when:
+    - network.value.config_nm | default(true) | bool
+  community.general.nmcli:
```

Comment:
Uhmmm, I'd say that `autoconnect` is maybe needed, not mandatory, but a nice to have.

Reply:
might be OK. Trying to find a reason not to have it, I don't see any, since we're able to NOT configure a connection with nmcli, like the current way.